src/cosign/mod.rs

```diff
@@ -262,20 +272,67 @@ Hr/+CxFvaJWmpYqNkLDGRU+9orzh5hI2RrcuaQ==
 
         let mut constraints: VerificationConstraintVec = Vec::new();
         let vc = CertSubjectEmailVerifier {
-            email: email_constraint.clone(),
-            issuer: Some(issuer.clone()),
+            email: wrong_email.clone(),
+            issuer: Some(issuer), // correct issuer
         };
         constraints.push(Box::new(vc));
 
         let vc = CertSubjectEmailVerifier {
-            email: email_constraint.clone(),
-            issuer: None,
+            email: wrong_email,
+            issuer: None, // missing issuer
```

Comment:
```suggestion
            issuer: None, // missing issuer - makes check more relaxed
```

The missing issuer per se, is not an error. This makes the verifier more relaxed. I've added the little clarification to ensure other readers don't assume this could lead to a verification == false

Reply:
True! done in https://github.com/sigstore/sigstore-rs/pull/42/commits/9f56fd64da567c8b0ebabf9d090717887e38f86f.